src/content/troubleshooting/faq/junit-report-usage.mdx

```diff
@@ -0,0 +1,119 @@
+
+## Report Structure and Key Data
+
+The Chromatic JUnit report is an XML file that summarizes the results of a build. The root `<testsuite>` tag contains high-level information about the entire test run. Key properties in every report include:
+
+- `buildNumber`: The Chromatic build number.
+- `buildUrl`: A direct link to the build results in the Chromatic UI.
+- `storybookUrl`: A link to your published Storybook.
+- `tests`: The total count of stories tested.
+- `failures`: The number of stories with visual changes that need review.
+- `skipped`: The number of stories that were skipped.
+- `errors`: The count of stories that encountered an error.
+
+Here's an example from a report file:
+
+```XML
+<testsuite name="Chromatic build 159" ... tests="86" failures="1" errors="0" skipped="0">
+    <properties>
+      <property name="buildNumber" value="159"/>
+      <property name="buildStatus" value="PENDING"/>
+      <property name="buildUrl" value="https://www.chromatic.com/build?appId=...&amp;number=159"/>
+      <property name="storybookUrl" value="https://...-nxjzlatrta.chromatic.com/"/>
+    </properties>
+</testsuite>
+```
+
+## Understanding Test Statuses
+
+Each <testcase> within the report represents a single Storybook story. Its status is indicated by the presence of child tags like `<failure>` or `<skipped>`.
+
+### Changed Stories
+
+If a story has visual changes that require review, it is marked with a `<failure>` tag.
+
+```XML
+<testcase classname="Atoms.Loader" name="Shadow Dom">
+  <properties>
+    <property name="result" value="PENDING" />
+  </properties>
+  <failure
+    message="Snapshot contains visual changes and must be reviewed"
+    type="PENDING"
+  />
+</testcase>
+```
+
+**Note:** If a story with changes in one build is run again in a subsequent build without being reviewed, it will still be flagged as "changed".
+
+### Skipped Stories
+
+Stories that are skipped during the build are explicitly marked with a `<skipped/>` tag.
+
+```XML
+<testcase classname="Atoms.Tooltip" name="Default ">
+  <skipped />
+</testcase>
+```
+
+### Passed and Disabled Stories
+
+A story that has passed successfully will appear as a simple `<testcase>` with no child tags.
+
+Stories that you've disabled in your Storybook configuration (`disabledSnapshot: true`) will still be listed in the report but will not be flagged with `<skipped/>`.
+
+## Storing the Report as an Artifact
+
+First, ensure you're saving the report in your CI environment. In GitHub Actions, you can use the `actions/upload-artifact` job. The report file typically follows the naming pattern `chromatic-build-*.xml`.
+
+```YAML
+# GitHub Actions Example
+- name: Upload Build Report
+  uses: actions/upload-artifact@v3
+  with:
+    name: chromatic-reports
+    path: chromatic-build-*.xml
+```
```

Comment:
The same applies here with the snippet, and also to align with the rest of the documentation, we can go with version 4 of the `upload-artifact` GitHub action
````suggestion
```yml title=".github/workflows/chromatic.yml"
- name: Upload Build Report
  uses: actions/upload-artifact@v4
  with:
    name: chromatic-reports
    path: chromatic-build-*.xml
```
````